tests/appsec/iast/sink/test_unvalidated_redirect_forward.py

```diff
@@ -49,11 +51,15 @@ class TestUnvalidatedForward:
     def setup_insecure_forward(self):
         self.sink_fixture_forward.setup_insecure()
 
+    @irrelevant(library="java", weblog_variant="resteasy-netty3", reason="there is no forward implementation")
+    @irrelevant(library="java", weblog_variant="jersey-grizzly2", reason="there is no forward implementation")
     def test_insecure_forward(self):
         self.sink_fixture_forward.test_insecure()
 
     def setup_secure_forward(self):
         self.sink_fixture_forward.setup_secure()
 
+    @irrelevant(library="java", weblog_variant="resteasy-netty3", reason="there is no forward implementation")
+    @irrelevant(library="java", weblog_variant="jersey-grizzly2", reason="there is no forward implementation")
```

Comment:
Actually there is, but we don't support it, right? If that's the case, then that'd be @missing_feature.